src/scripts/create_repos.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+
+repo_path=$1
+
+USHIFT_LOCAL_REPO_FILE=/etc/yum.repos.d/microshift-local.repo
+OCP_MIRROR_REPO_FILE=/etc/yum.repos.d/openshift-mirror-beta.repo
+
+    cat > "${USHIFT_LOCAL_REPO_FILE}" <<EOF
+[microshift-local]
+name=MicroShift Local Repository
+baseurl=${repo_path}
+enabled=1
+gpgcheck=0
+skip_if_unavailable=0
+EOF
+
+    cat > "${OCP_MIRROR_REPO_FILE}" <<EOF
```

Comment:
Can we open a JIRA issue for getting rid of this dependency?
We should be able to pull all the dependencies without the mirror.
See [this](https://github.com/openshift/microshift/blob/main/scripts/devenv-builder/configure-vm.sh#L239) for example

Reply:
issue created https://issues.redhat.com/browse/USHIFT-5687